addons/aircraft/CfgAmmo.hpp

```diff
@@ -50,16 +50,27 @@ class CfgAmmo {
         timeToLive = 40;
     };
 
-    // helper projectiles to simulate a rof > fps
-    class ACE_Gatling_30mm_HE_Plane_CAS_01_Deploy: Gatling_30mm_HE_Plane_CAS_01_F {
-        simulation = "shotSubmunitions";
-        triggerTime = 0;
-        submunitionAmmo = "ACE_Gatling_30mm_HE_Plane_CAS_01_Sub";
-        submunitionConeType[] = {"custom", {{0,0}, {0,0}, {0,0}} };
+    // GAU-8 Mix ammo (AP&HE)
+    class ACE_GAU8_30mm_Mix_Plane_CAS_01_F: Gatling_30mm_HE_Plane_CAS_01_F {
+        laserLock = 1;
+        hit = 300;
+        indirectHit = 100;
+        indirectHitRange = 3.5;
```

Comment:
I tested it and it's not as bad as I feared. Maybe it can be tweaked later to try to make it less destructive to buildings, but for now it's okay as far as I can tell.